client/src/components/Detail.js

```diff
@@ -20,10 +21,39 @@ const Detail = ({ match }) => {
   };
 
   return (
-    <div>
-      <h1>
-        <img src={comic.cover} alt="cover" />
-      </h1>
+    <div className={styles["detail"]}>
+      <div className={styles["detail__col1"]}>
+        <div className={styles["detail__cover"]}>
+          <img src={comic.cover} alt="cover" />
+        </div>
+      </div>
+      <div className={styles["detail__col2"]}>
+        <div className={styles["detail__cover"]}>
+          <h1 className={styles["detail__title"]}>{comic.name}</h1>
+          <p>
+            <strong>Publisher:</strong> {comic.publisher}
```

Comment:
We should avoid using `<strong>` tags unless we want to call particular attention to this element - screen readers will interpret this as meaning this element is very important to the meaning of the page. I'd just use a <div> or a <p> and apply the styles you want using css so we have more fine control - we'll want to use a constant typography style for this at some point anyway.